scripts/test-functions-deploy.js

```diff
@@ -25,9 +25,6 @@ var functionsConfig = require("../lib/functionsConfig");
 
 var clc = require("cli-color");
 var firebase = require("firebase");
```

Comment:
I'm fairly certain that `firebase` in the root `package.json` is really, really, really, really old - it would be great if it was updated. (This file is the _only_ requirement for having `firebase` as a dependency - I've looked - so since you're updating it...)